<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>934</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log Dialog</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QFrame" name="grp_searchInfo">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>TR_ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="trEdit">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>IP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ipEdit">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>GUID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edt_guid">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchBtn">
           <property name="text">
            <string>실행</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_tree_view">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QWidget" name="grp_logTree" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="twLine">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="placeholderText">
               <string>Search...</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="twBtn">
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="collpsBtn">
              <property name="text">
               <string>▲</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="expndBtn">
              <property name="text">
               <string>▼</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeWidget" name="tw_logTree">
            <column>
             <property name="text">
              <string notr="true">로그트리</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Line</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Time</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>프로그램명</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>수행시간</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="grp_logView" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="grp_search">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLineEdit" name="edt_searchLogView">
                 <property name="font">
                  <font>
                   <family>맑은 고딕</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="placeholderText">
                  <string>Search...</string>
                 </property>
                 <property name="clearButtonEnabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_findPrevious">
                 <property name="text">
                  <string>Pevious</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_findNext">
                 <property name="text">
                  <string>Next</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="errBtn">
                 <property name="text">
                  <string>(E)</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QCheckBox" name="re">
                 <property name="text">
                  <string>Reqular expressions</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="cs">
                 <property name="toolTip">
                  <string>Case sensitive</string>
                 </property>
                 <property name="text">
                  <string>Match Case</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="wo">
                 <property name="toolTip">
                  <string>Whole Words</string>
                 </property>
                 <property name="text">
                  <string>Words</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="wrap">
                 <property name="text">
                  <string>Wrap around</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="show_">
                 <property name="text">
                  <string>Unfold folded text</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="posix">
                 <property name="toolTip">
                  <string>POSIX-compatible RE</string>
                 </property>
                 <property name="text">
                  <string>POSIX</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="log_layout"/>
          </item>
          <item>
           <widget class="QFrame" name="">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="curcnt">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>맑은 고딕</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::TabFocus</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <family>맑은 고딕</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>/</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="totcnt">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>맑은 고딕</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pageBtn">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>&gt;&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>trEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>1522</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edt_searchLogView</sender>
   <signal>returnPressed()</signal>
   <receiver>btn_findNext</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1070</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>1419</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ipEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>867</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>1496</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edt_guid</sender>
   <signal>returnPressed()</signal>
   <receiver>searchBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1223</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>1519</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
